styles/components/Block.scss

```diff
@@ -0,0 +1,5 @@
+.Block {
+  div > span > p {
+    line-height: 1.5rem;
```

Comment:
what is `Block`? why do we need this?

Reply:
@mokaymokay it was my attempt to give even line-height for paragraphs in Case Study pages from previous PR. Should have been removed, now it is removed. Thanks for pointing this out ;)